### PR TITLE
Integrate with importfile and download extension

### PR DESCRIPTION
**Pull request by HunseopJeong**
- Renamed the "importfile" to the "file-extension"
- Move download command to the file-extension

Signed-off-by: Hunseop Jeong <hs85.jeong@samsung.com>

### CODE REVIEW

File: libs/brackets-server/embedded-ext/file-extension/main.js [hunk 1]
```diff
@@ -292,39 +291,57 @@ define(function (require, exports, module) {
         }
     }
 
+    function handleDownloadFile() {
+        var selected = ProjectManager.getSelectedItem();
+
+        if (selected && selected.isFile) {
+            var projectId = PreferencesManager.getViewState("projectId");
```
Comment (Kwanghyuk-Kim) [inline libs/brackets-server/embedded-ext/file-extension/main.js]: var -> const ?

File: libs/brackets-server/embedded-ext/file-extension/main.js [hunk 2]
```diff
@@ -292,39 +291,57 @@ define(function (require, exports, module) {
...
+            var filePath = brackets.app.convertFilePathToServerPath(selected.fullPath, projectId);
```
Comment (Kwanghyuk-Kim) [inline libs/brackets-server/embedded-ext/file-extension/main.js]: ditto,

File: libs/brackets-server/embedded-ext/file-extension/main.js (hunk 1 above)
Comment (Kwanghyuk-Kim) [reply libs/brackets-server/embedded-ext/file-extension/main.js]: did you run grunt htmllint ?
IMO, need to consider this as new file.

**Comment by Kwanghyuk-Kim** [comment]
Did you run grunt htmllint and csslint for this commit ?
Let's fix css and htmllint as moved files can be considered as new files.

File: libs/brackets-server/embedded-ext/file-extension/main.js (hunk 1 above)
Comment (HunseopJeong) [reply libs/brackets-server/embedded-ext/file-extension/main.js]: Done.

File: libs/brackets-server/embedded-ext/file-extension/main.js (hunk 2 above)
Comment (HunseopJeong) [reply libs/brackets-server/embedded-ext/file-extension/main.js]: Done.

**Comment by HunseopJeong** [comment]
Fixed. I think we should fix all html and css rule issue.
So I will make new issue to fix it.

File: libs/brackets-server/embedded-ext/file-extension/main.js [hunk 3]
```diff
@@ -292,39 +291,57 @@ define(function (require, exports, module) {
         }
     }
 
+    function handleDownloadFile() {
+        const selected = ProjectManager.getSelectedItem();
+
+        if (selected && selected.isFile) {
+            const projectId = PreferencesManager.getViewState("projectId");
+            const filePath = brackets.app.convertFilePathToServerPath(selected.fullPath, projectId);
+            const anchor = document.createElement("a");
```
Comment (Kwanghyuk-Kim) [inline libs/brackets-server/embedded-ext/file-extension/main.js]: const type would be fine ?
anchor.herf seems to be modified.

Comment (HunseopJeong) [reply libs/brackets-server/embedded-ext/file-extension/main.js]: Yes. 'anchor' is the object.
Object is reference type. So we can change member value of the 'anchor'.
(https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Statements/const)

**Pull request merged**